pkgs/epnix/tools/phoebus/setup-hook/setup-hook.sh

```diff
@@ -23,5 +26,10 @@ installPhoebusJar() {
 	makeWrapper "@jdk@/bin/java" "$out/bin/$name" \
 		--add-flags "-classpath $out/share/java/$jarName:$depsPath/*" \
 		--add-flags '${JAVA_OPTS}' \
-		--add-flags "$mainClass"
+		--add-flags "$mainClass"	\
+		--set-default JAVA_XMS "1G" \
+		--set-default JAVA_XMX "4G" \
+		--add-flags '-Xms$JAVA_XMS' \
+		--add-flags '-Xmx$JAVA_XMX' \
```

Comment:
Hmmm, since this is the setup-hook for all Phoebus code, this will apply to Phoebus client, but also alarm server, etc.

I think we should have those higher values only for Phoebus client, since we want to be able to have the whole Phoebus ecosystem on small servers.

So we can do it like this:

```suggestion
		--add-flags "$java_runtime_opts" \
```

And then in the phoebus client package, you can add `java_runtime_opts = "-Xmx4G"`